src/install-sfdx/commands/setup.yml

```diff
@@ -1,19 +1,20 @@
 description: |
   Install `sfdx` CLI and plugins.
 parameters:
+  package:
+    description: The sfdx CLI to install (npm package like "name@version")
+    type: string
+    default: sfdx-cli
   plugins:
-    description: A comma-delimited list of sfdx plugins to install
+    description: A comma-delimited list of sfdx plugins to install (npm packages like "name@version")
     type: string
     default: ""
 steps:
-  - restore_cache:
-      keys:
-        - orb-install-sfdx-v1
```

Comment:
Is this being removed because `package` may change? Could factor that into the cache key, if needed.

Reply:
`package` being specified does not mean a static set of dependencies… for example, the default `"sfdx-cli"` always picks up the latest. Also, the `plugins` parameter value adds another dynamic set of package dependencies. 

To be effective, the cache key would need to follow the convention for npm projects and use the hash of `package-lock.json`, which can be calculated before the installation commands run.

I'm keen on solving this, but it seems to me that the `sfdx evergreen` installation process simply doesn't support us pre-calculating such a singular uniqueness hash, without first performing the whole installation.